DeploymentCloud/Deployment.DataX/common.parameters.txt

```diff
@@ -20,6 +20,9 @@ subscriptionId=
 # ResourceGroupName to generate resources 
 resourceGroupName=DataX
 
+# y if you want to setup Cosmos Db output sinks for samples
+setupAdditionalResourcesForSample=n
```

Comment:
should this default value be y? Will the samples still work w/o resources when it is n?

Reply:
Is Cosmos DB only one to be added?

Reply:
default will be n, so that we dont spin up new resources. Sample will work without new resources. Currently the only new resource is CosmosDb but in future we will be adding SQL db/server as well